docs/UserGuide.md

```diff
@@ -45,8 +45,45 @@ Apart from being your all-encompassing address book, NeoBook also has an Events
 [Back To Contents](#table-of-contents)
 <hr style="border:2px solid gray">
 
+## How to read the User Guide
+
+<div markdown="block" class="alert alert-info">
+
+####  :information_source: Notes about the command format:<br>
+
+* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+
+* Items in square brackets are optional.<br>
+  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+
+* Items with `…`​ after them can be used multiple times including zero times.<br>
+  e.g. `t/TAG…​` means that we can exclude the tag prefix completely or use it as `t/friend`, `t/friend t/family` etc.
+
+* Items in `{ }` and have a `...` followed after them must be used at least once, and can be used multiple times.<br>
+  e.g. For `editevent INDEX {PREFIX/PARAMETER}...` will allow `editevent 1 r/Weekly` or `editevent 2 d/Do work r/Daily`, but not `editevent 1`.
```

Comment:
Perhaps the ellipsis can be standardised as `...`

Also, this new format is risky as the usage messages in NeoBook is still using the `[ ]` format. If we decide to use `{ }`, then it should be consistent throughout the UG and NeoBook app (from a quick glance, `edit` and `find` in this PR are not still following `{ }`)